cmd/livepeer/livepeer.go

```diff
@@ -98,6 +98,7 @@ func main() {
 	broadcaster := flag.Bool("broadcaster", false, "Set to true to be a broadcaster")
 	orchSecret := flag.String("orchSecret", "", "Shared secret with the orchestrator as a standalone transcoder")
 	transcodingOptions := flag.String("transcodingOptions", "P240p30fps16x9,P360p30fps16x9", "Transcoding options for broadcast job")
+	maxAttempts := flag.Int("maxAttempts", 3, "How many times to attempt a transcode")
```

Comment:
Might "Maximum number of transcode attempts" be more descriptive here?

Reply:
Updated to language to "Maximum transcode attempts" in 8e12347944a0460ee298acca2fbd02b1772915f1